### ISSUE TITLE
Edit Address - Accessibility issues

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16395]
## What Happened?
**VoiceOver / TalkBak**

1. With VoiceOver enabled on iOS, the country, state, and APO/FPO/DPO drop down fields are not announced as pickers when selected
2. With TalkBack enabled on Android, the country, state, and APO/FPO/DPO drop down fields are not announced as selectable (i.e. TalkBack does not say "double-tap to activate")
3. With VoiceOver enabled on iOS, the save button has multiple selections
4. With VoiceOver and TalkBack enabled on iOS and Android, the cancel button a11yHint is not being read
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)
5. Checkbox bug on Android is covered in a [separate ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16484)

**Keyboard**

6. With Keyboard linked to Android, the keyboard focus gets trapped in the Country field and the other elements can not be accessed

**Resize**

7. When a larger font/display size is enabled on iOS and Android, the field labels are incorrectly formatted during resizing
8. When a larger font/display size is enabled on iOS and Android, the hint labels are cut off
9. When a larger font/display size is enabled on iOS, the hint labels are resized inconsistently 
10. Resizing of header content bug is covered in a [separate ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/15534)
    - Note: This affects the save button on iOS and Android

## Specs:
- Device: Android - LG LM-Q720 Stylo 5 with TalkBack enabled
- Device: iOS - iPhone XR with VoiceOver enabled
- Bluetooth Keyboard

## Steps to Reproduce
**VoiceOver / TalkBak**

For 1: 
1. With VoiceOver enabled on iOS, navigate to either the mailing or residential address page
2. Select either the country, state, or APO/FPO/DPO field
3. Note that VoiceOver reads out the text of these fields but does not note them as pickers

For 2:
1. With TalkBack enabled on Android, navigate to either the mailing or residential address page
2. Select either the country, state, or APO/FPO/DPO field
3. Note that TalkBack does not read the fields as selectable (does not say "double-tap to activate")

For 3:
1. With VoiceOver enabled on iOS, navigate to either the mailing or residential address page
2. Note that the save button is disabled
3. Press the save button. Note that the VoiceOver focus is only on the save text and VoiceOver only reads "save"
4. Press the save button again. Note that the VoiceOver focus expands and VoiceOver reads "Save. Dimmed. Button. Saves Changes"

**Keyboard**

For 6:
1. With Keyboard linked to Android, navigate to either the mailing or residential address page
2. Press tab or the down arrow key. The cancel button will be in focus
3. Press tab or the down arrow key. The checkbox element will be in focus 
4. Press tab or the down arrow key. The Country field will be in focus
5. Press tab or the down arrow key. The Country field will remain (trapped) in focus 

**Resize**

For 7:
1. On iOS and Android enable a larger font/display size
2. Navigate to either the mailing or residential address page
3. Note field labels are incorrectly formatted (cut off by letter)

For 8:
1. On iOS and Android enable a larger font/display size 
2. Navigate to either the mailing or residential address page
3. Note the hint labels are getting cut off

For 9:
1. On iOS enable a larger font/display size
2. Navigate to either the mailing or residential address page
3. Note the inconsistent hint label resizing

## Desired Behavior
**VoiceOver / TalkBak**

For 1:
- With VoiceOver enabled on iOS, the country, state, or APO/FPO/DPO fields will be read out as pickers 

For 2:
- With TalkBack linked to Android, the country, state, or APO/FPO/DPO fields will be read out as selectable (TalkBack will say "double-tap to activate")

For 3:
- With VoiceOver enabled on iOS, the save button will have one selection state 

**Keyboard**

For 6:
- With Keyboard linked to Android, the keyboard will not get trapped on the County drop-down field and all the other elements on the page will be able to be focused

**Resize**

For 7:
- On iOS and Android, when a larger font/display size is enabled, the field label will not be cut off by letter 
     - Note: Will have to recheck expected behavior

For 8:
- On iOS and Android, when a larger font/display size is enabled, the hint labels will be cut off

For 9:
- On iOS, when a larger font/display size is enabled, the hint labels will be resized correctly and consistently 

## Acceptance Criteria
- See desired behavior

## Bug Severity
- 3 - Medium

## Linked Story (s)
- Edit US address #13893
- Edit military base address #15058
- Edit international address #15059

## Screen Shot (s) and Additional Information
7. Field label resizing bug 
On Android 🔽
![Edit Address - Android Field Label.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/b39c2b3e-c071-4033-9bac-c14fe00e2089)
On iOS 🔽
![Edit Address - iOS Label.PNG](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/2f86ce41-25fc-4e63-8f11-e3252dcb61d6)

8. Hint label resizing bug 
- On Android 🔽
![Edit Address - Android Label.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/ebb2ac94-022b-450b-98da-69c80f4623ad)
- On iOS 🔽
![Edit Address - iOS Hint Label.PNG](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/c3e3a0c2-4db6-4789-9742-a6576e02877c)

9. iOS inconsistent hint label resizing bug 🔽
![Edit Address - iOS Hint Label Resize.PNG](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/b6b31606-4611-4bea-8307-403abbc97431)

10. Resizing bug is covered in a [separate ticket](spaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/15534)). Included how it affects edit address pages 
- Android view of bug 🔽
![Edit Address - Android Save Button.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/095d5d55-57c3-4efb-8aaf-07452c7bf49d)
- iOS view of bug 🔽
![Edit Address - iOS Save Button.PNG](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/a1797a38-061e-4b9a-9a04-f67e38a3ca58)

**Comment by digitalken** [comment]
PASS 5/19/2021 - Review of issues.  Now passes as a result of changes made to app.  Closing.
